packages/ui/src/memberships/components/MemberRoles.tsx

```diff
@@ -85,6 +96,32 @@ export const MemberRoles = ({ size, max, wrapable, roles }: MemberRolesProps) =>
   )
 }
 
+export interface MemberRolePopupContentProps {
+  roles: MemberRole[]
+}
+
+export const MemberRolePopupContent = ({ roles }: MemberRolePopupContentProps) => {
+  const groupAddress = `#/working-groups/${groupNameToURLParam(roles[0].groupName)}`
```

Comment:
It might be worth checking if this can't be constructed using react-router utils. Because we might change the routing creation mechanism and then `#` will break that.